src/genie/libs/parser/iosxe/show_lisp.py

```diff
@@ -3412,3 +3412,170 @@ def filter_lines(raw_output, remove_lines):
         return lisp_site_dict
 
 
+
+
+# ================================================
+# Schema for:
+#  * 'show lisp eid-table vrf {vrf} ipv4 map-cache'
+# ================================================
+class ShowLispEidTableVrfUserIpv4MapCacheSchema(MetaParser):
+    """Schema for show lisp eid-table vrf {vrf} ipv4 map-cache."""
+
+    schema = {
+        "vrf": str,
+        "iid": int,
+        "number_of_entries": int,
+        "eid": {
+            str: {
+                "subnet": str,
+                "uptime": str,
+                "expire": str,
+                "source": list,
+                "rloc": {
+                    Optional("status"): str,
+                    Optional("action"): str,
+                    Optional("ip"): str,
+                    Optional("uptime"): str,
+                    Optional("state"): str,
+                    Optional("priority"): int,
+                    Optional("weight"): int,
+                    Optional("encap_iid"): str
+                }
+            }
+        }
+    }
+
+
+# ================================================
+# Parser for:
+#  * 'show lisp eid-table vrf User ipv4 map-cache'
+# ================================================
+class ShowLispEidTableVrfUserIpv4MapCache(ShowLispEidTableVrfUserIpv4MapCacheSchema):
+    """Parser for show lisp eid-table vrf {vrf} ipv4 map-cache"""
+
+    cli_command = 'show lisp eid-table vrf {vrf} ipv4 map-cache'
+
+    def cli(self, vrf="", output=None):
```

Comment:
I think ```vrf``` should not have default value, it's a mandatory argument, so please remove ```=""```
```
def cli(self, vrf, output=None)

Reply:
Updated! Thanks!